examples/README.md

```diff
@@ -2,9 +2,16 @@
 
 # Examples
 
-This directory contains fully functional code examples that you can use to learn more about Objax
+This directory contains fully functional code examples that you can use to learn more about Objax.
 
-See:
-* [Image Classification](classify/img/README.md)
+Examples from classic machine learning:
+* [Image Classification](image_classification/README.md)
```

Comment:
In image classification, I think `horses_logistic.py` should be renamed.
I think it would be better either to remove name of the class, i.e. use `logistic.py` or `logistic_regression.py`
Or include names of both classes. i.e. `horses_or_humans_logistic.py`

For all other files we use dataset name (`mnist_dnn.py`, `imagenet_...`), which is inconsistent with `horses_logistic.py`

Reply:
Done.